packages/firebase/index.d.ts

```diff
@@ -7054,6 +7136,8 @@ declare namespace firebase.database {
     logger?: boolean | ((a: string) => any),
     persistent?: boolean
   ): any;
+
+  export type EmulatorMockTokenOptions = firebase.EmulatorMockTokenOptions;
```

Comment:
I think we can just use `firebase.EmulatorMockTokenOptions` without needing the alias

Reply:
We already released a version with `firebase.firestore.EmulatorMockTokenOptions` and I think we should probably match that for `database` for consistency. WDYT?

Reply:
hmm, good point. Ok, let do that.